src/Symfony/Component/Lock/Store/MongoDbStore.php

```diff
@@ -268,7 +268,7 @@ public function exists(Key $key): bool
                 '$gt' => $this->createMongoDateTime(microtime(true)),
             ],
         ], [
-            'readPreference' => new ReadPreference(\defined(ReadPreference::PRIMARY) ? ReadPreference::PRIMARY : ReadPreference::RP_PRIMARY),
```

Comment:
The existing code is incorrect: it checks the existence of the constant **value**. See the conversation why this condition was added: https://github.com/symfony/symfony/pull/37140#discussion_r437805079

This was removed in 6.4 by https://github.com/symfony/symfony/commit/bc24cb3db7d8e36785e00c5ff1e4e7467ebce83a#diff-d0f7d4ac57862532ef468d4fa38f1879eeb1c6e6f142118ecfa28dba891601b4L266